src/rp2_common/pico_lwip/lwip.BUILD

```diff
@@ -2,18 +2,36 @@ load("@pico-sdk//bazel:defs.bzl", "incompatible_with_config")
 
 package(default_visibility = ["//visibility:public"])
 
+# Some of the LWIP sys_arch.h and the lwip headers depend circularly on one
+# another. Include them all in the same target.
+cc_library(
+    name = "pico_lwip_headers",
+    hdrs = glob(["**/*.h"]),
+    includes = [
+        "contrib/ports/freertos/include/arch",
+        "src/include",
+    ],
+    deps = [
+        "@pico-sdk//bazel/config:PICO_LWIP_CONFIG",
+        "@pico-sdk//src/rp2_common/pico_lwip:pico_lwip_config",
+    ],
+)
```

Comment:
Can the visibility be constrained to `visibility = ["//visibility:private"],` or does FreeRTOS need to be able to directly access this library?

Reply:
Actually I'm realizing that since this is bzlmod, FreeRTOS shouldn't be able to see `@lwip` directly at all, so it should be safe to add a visibility constraint.

Reply:
Done.